ApplicationCode/ProjectDataModel/RimGeoMechCase.cpp

```diff
@@ -834,6 +940,15 @@ void RimGeoMechCase::reloadSelectedElementPropertyFiles()
     }
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimGeoMechCase::importElementPropertyFile()
+{
+    QAction* importAction = caf::CmdFeatureManager::instance()->action( "RicImportElementPropertyFeature" );
```

Comment:
Create a static function in RicImportElementPropertyFeature, and call this function. The action() function is intended to be used if the action will be displayed in a menu.

Reply:
Fixed.